docs/introduction/CORE_CONCEPTS.mdx

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I think we can pitch microfrontends here more aggressive. Let's get some hype

Reply:
Sure, this PR isn't about the content, but rather the structure.

Reply:
To answer your comment though, I'm all for explaining the motivation behind Loom in the docs, but let's keep them concise, we're not writing a book here :)

Reply:
Hehe, sure :)